string[] camera_paths
---
bool success
